aim/web/ui/src/pages/Params/components/ParamsTableGrid/ParamsTableGrid.tsx

```diff
@@ -146,18 +146,74 @@ function getParamsTableColumns(
       const systemMetricsList: ITableColumn[] = [];
       const metricsList: ITableColumn[] = [];
       Object.keys(metricsColumns[key]).forEach((metricContext) => {
-        const columnKey = `${systemMetric ? key : `${key}_${metricContext}`}`;
+        const contextName = metricContext ? `_${metricContext}` : '';
+        const columnKey = `${systemMetric ? key : `${key}${contextName}`}`;
+        const sortValueKey = `metrics.${key}${contextName}`;
+        let index = -1;
+        const sortItem: SortField | undefined = sortFields?.find((value, i) => {
```

Comment:
maybe we can simplify this code part, u can use here the `findIndex` method to find the `sortField` index.
 
`const sortItemIndex = sortFields((field, i) => field.value === sortValueKey);`
`const sortItem = sortFields[sortItemIndex];`